docs/source/en/optimization/memory.md

```diff
@@ -386,48 +458,36 @@ class TracedUNet(torch.nn.Module):
         sample = unet_traced(latent_model_input, t, encoder_hidden_states)[0]
         return UNet2DConditionOutput(sample=sample)
 
-
-pipe.unet = TracedUNet()
+pipeline.unet = TracedUNet()
 
 with torch.inference_mode():
     image = pipe([prompt] * 1, num_inference_steps=50).images[0]
 ```
 
 ## Memory-efficient attention
```

Comment:
I think this only contributes to memory but also latency. Could we make a note? I think it might be a bit confusing for readers given we mention SDPA in the inference acceleration guide and mention memory-efficient attention here.